src/shared/models/k8sResource.ts

```diff
@@ -1,36 +1,24 @@
-import {Scalar} from 'yaml';
+import * as Rt from 'runtypes';
```

Comment:
is runtypes new? I'd recommend Zod as my go to schema validator as it works tremendously well with TypeScript.